### PR TITLE
Translation

### LINKED ISSUE #82: Translation
## Summary

translate the website into Turkish or Arabic language.

### PR DESCRIPTION
**Pull request by batoulst**
# Description

- installing i18n and using it for translation
- changing the direction of html right to left when language changes to Arabic

- new locales folder that contains the translation(ar,en,tr)

- en and ar files are finished
- tr file is initialized but needs tr translation
- translating home page, navbar, footer with Arabic is done

- there is bugs with design and css

**Note: make npm install before running**

## Related Issue

Fixes #82 

## Screenshots

image
image

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Tested locally
- [ ] Added tests

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] I've run linter, prettier, and removed all `console.log`
- [x] I've reverted all the files back that are not related to the issue (App.js, index.js, ...)

### CODE REVIEW

**Review by ahmadmhd357 — APPROVED** [review]
it looks amazing and a lot of work and somehow I feel that we have cleaner code now, thank you so much

**Comment by ahmadmhd357** [comment]
> it looks amazing and a lot of work and somehow I feel that we have cleaner code now, thank you so much

But it seems to have some styling issue that we can work on later

File: src/components/LanguageSelect.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+          className="flex items-center justify-between font-medium h-10 border border-cyan-600 rounded-full px-4  w-40 cursor-pointer"
+        >
+          <option value="en" className="cursor-pointer">
+            English
+          </option>
+          <option value="tr" className="cursor-pointer">
+            Türkçe
+          </option>
+          <option value="ar" className="cursor-pointer">
+            العربية
```
Comment (Omer-Amr) [inline src/components/LanguageSelect.jsx:33]: Feeling like we have wonderful results upcoming 🤩

File: src/locales/ar.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    },
+    "location": {
+      "text1": "\"كان العمل مع الفريق في ريفيوبوك تجربة مجزية بشكل لا يصدق. إنه لمن دواعي سرورنا أن نرى التأثير الإيجابي الذي نحدثه على حياة اللاجئين كل يوم.\"",
+      "text2": "\"فخور بأن أكون جزءًا من فريق ريفيوبوك وأنا معجب باستمرار بتعاطف واحتراف زملائي. إنه لمن دواعي سروري العمل مع هذه المجموعة المتفانية من الأفراد الملتزمين بإحداث تغيير في حياة اللاجئين\"",
+      "text3": "\"بصفتي عضوًا في فريق ريفيوبوك ، فأنا دائمًا مندهش من تأثير عملنا على حياة اللاجئين. إنه لمن دواعي سرورنا حقًا أن أكون جزءًا من هذه المنظمة.\"",
+      "text4": "\"سمحت لي كوني جزءًا من فريق ريفيوبوك بإحداث تغيير ملموس في حياة اللاجئين. إنه لشرف كبير أن أكون قادرًا على المساهمة في مثل هذه القضية المهمة والضرورية.\"",
+      "team": "عضو فريق ريفيوبوك"
+    }
+  }
+}
```
Comment (Omer-Amr) [inline src/locales/ar.json:57]: Awsome !

**Review by Omer-Amr — APPROVED** [review]
Realy appreciate your work!
we can make some changes on text later thank you

**Pull request merged**